wsapp_connector.py

```diff
@@ -262,6 +276,150 @@ def _handle_get_events(self, param):
         # For now return Error with a message, in case of success we don't set the message, but use the summary
         # return action_result.set_status(phantom.APP_ERROR, "Action not yet implemented")
 
+    def _handle_poll_events(self, param):
+        # Implement the handler here
+        # use self.save_progress(...) to send progress messages back to the platform
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+        # Add an action result object to self (BaseConnector) to represent the action for this param
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        # Access action parameters passed in the 'param' dictionary
+
+        # Required values can be accessed directly
+        # required_parameter = param['required_parameter']
+
+        # Optional values should use the .get() function
+
+        # make rest call
+        client_id = self._client_id
+        client_secret = self._client_secret
+        base_url = self._base_url
+        token = self._get_token(client_id, client_secret, base_url)
+        headers = {'Authorization': f'Bearer ' + token}
+        ret_val, response = self._make_rest_call(
+            'whoami/v1/whoami', action_result, params=None, headers=headers
+        )
+        organizationId = response["organizationId"]
+        info = self.get_container_info(container_id=224)
```

Comment:
Will this hardcoded id always remain the same?

Reply:
Currently it needs to be hardcoded.